composer.json

```diff
@@ -12,14 +12,13 @@
     "require": {
         "php": "^7.2",
         "symfony-cmf/resource-bundle": "^1.0",
-        "jms/serializer-bundle": "^1.0 | ^2.0 | ^3.0",
+        "jms/serializer-bundle": "^3.0",
```

Comment:
i would rather have the conflict entry than dropping jms/serializer-bundle 2 support without any other reasons. if we can be more flexible, it is better for the users as we don't force them to upgrade.

Reply:
Serializer-bundle 2 has JMS serializer v1. The reason for dropping JMS serializer v1 is to do bundle compatible with other bundles which required v2 with none BC to v1. If you need v1 then you can decrease this bundle version in the composer file to 1.1. That what I think.

Reply:
but the only issue seems to be that we need jms serializer 1.2 or better, not that jms serializer 1 does not work. when there is no cost (as in complicated code to support the two versions) in allowing it, i'd rather keep allowing serializer bundle 2.